framework/console/Request.php

```diff
@@ -66,10 +66,15 @@ public function resolve()
 
         $params = [];
         foreach ($rawParams as $param) {
-            if (preg_match('/^--(\w+)(=(.*))?$/', $param, $matches)) {
+            if (preg_match('/^--(\w+)(?:=(.*))?$/', $param, $matches)) {
                 $name = $matches[1];
                 if ($name !== Application::OPTION_APPCONFIG) {
-                    $params[$name] = isset($matches[3]) ? $matches[3] : true;
+                    $params[$name] = isset($matches[2]) ? $matches[2] : true;
+                }
+            } elseif (preg_match('/^-(\w+)(?:=(.*))?$/', $param, $matches)) {
+                $name = $matches[1];
+                if ($name !== Application::OPTION_APPCONFIG) {
```

Comment:
just remove it, it does not make sense in the alias branch. it is already in line 71 and should only apply to `--` parameters.